### PR TITLE
Fix link on /smartstart/guide/hardware-setup

### LINKED ISSUE #8647: Link points to google docs
The link to "Canonical’s device enablement" (first paragraph, last sentence) points to a gdoc copy rather than "https://ubuntu.com/smartstart/guide/device-enablement".
---
*Reported from: https://ubuntu.com/smartstart/guide/hardware-setup*

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Fix link on /smartstart/guide/hardware-setup

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/smartstart/guide/hardware-setup
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that 'Canonical’s device enablement' now links to the device enablement page.

## Issue / Card

Fixes #8647

### CODE REVIEW

File: templates/smartstart/guide/hardware-setup.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ context:
...
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------ | --------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------- |
```
Comment (anthonydillon) [inline templates/smartstart/guide/hardware-setup.md:18]: Do you need all of this?

**Review by anthonydillon — APPROVED** [review]
+1 with a question

File: templates/smartstart/guide/hardware-setup.md (hunk 1 above)
Comment (pmahnke) [reply templates/smartstart/guide/hardware-setup.md:18]: Oh, it is a markdown table prettier thing, obviously it isn't helping in this case, but quiet useful most of the time. I assume you want me to uninstall it from my code editor and remove all the space?

File: templates/smartstart/guide/hardware-setup.md [hunk 2]
```diff
@@ -8,15 +8,15 @@ context:
...
+| amd64                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | i386         | arm64                                                                       | armhf                                                                                    |
```
Comment (pmahnke) [inline templates/smartstart/guide/hardware-setup.md:17]:
```suggestion
| amd64|i386|arm64|armhf|
```

File: templates/smartstart/guide/hardware-setup.md (hunk 1 above)
Comment (pmahnke) [inline templates/smartstart/guide/hardware-setup.md:18]:
```suggestion
| --- | --- | --- | --- |
```

File: templates/smartstart/guide/hardware-setup.md [hunk 3]
```diff
@@ -8,15 +8,15 @@ context:
...
+| [Intel NUC7CJYH](https://certification.ubuntu.com/hardware/201805-26256)<br> [Intel NUC7PJYH](https://certification.ubuntu.com/hardware/201805-26252)<br> [Intel NUC7i3DNHE](https://certification.ubuntu.com/hardware/201802-26085)<br> [Intel NUC7i5DNHE](https://certification.ubuntu.com/hardware/201802-26086)<br> [Intel NUC7i5DNHE](https://certification.ubuntu.com/hardware/201802-26087)<br> [Intel TANK-870-Q170](https://certification.ubuntu.com/hardware/201808-26450) | Intel Edison | Qualcomm Dragonboard 410c<br> Raspberry Pi model 3<br> Raspberry Pi model 4 | Raspberry Pi model 2<br>Raspberry Pi model 3<br>Raspberry Pi model 4<br>Raspberry Pi CM3 |
```
Comment (pmahnke) [inline templates/smartstart/guide/hardware-setup.md:19]:
```suggestion
|[Intel NUC7CJYH](https://certification.ubuntu.com/hardware/201805-26256)<br> [Intel NUC7PJYH](https://certification.ubuntu.com/hardware/201805-26252)<br> [Intel NUC7i3DNHE](https://certification.ubuntu.com/hardware/201802-26085)<br> [Intel NUC7i5DNHE](https://certification.ubuntu.com/hardware/201802-26086)<br> [Intel NUC7i5DNHE](https://certification.ubuntu.com/hardware/201802-26087)<br> [Intel TANK-870-Q170](https://certification.ubuntu.com/hardware/201808-26450) | Intel Edison  | Qualcomm Dragonboard 410c<br> Raspberry Pi model 3<br> Raspberry Pi model 4| Raspberry Pi model 2<br>Raspberry Pi model 3<br>Raspberry Pi model 4<br>Raspberry Pi CM3|
```

**Pull request merged**